rs/bitcoin/ckbtc/minter/src/state.rs

```diff
@@ -463,8 +459,8 @@ impl CkBtcMinterState {
             min_confirmations,
             mode,
             new_kyt_principal,
-            kyt_principal,
             kyt_fee,
+            ..
```

Comment:
nit: this I think I would not add. I think it's a nice feature that if something changes in `UpgradeArgs`, this place will not compile and one needs to actively need to think of the changes that are required in that method.